test/rsql-filter-builder.test.ts

```diff
@@ -8,7 +8,7 @@ describe('RSQLFilterBuilder', () => {
       .column('blah')
       .equalTo('123')
       .toList();
-    expect(list.build()).toEqual(`blah==${encodeURIComponent('"123"')}`);
+    expect(list.build()).toEqual(`blah=in=${encodeURIComponent('"123"')}`);
```

Comment:
Does the `=in=` syntax allow a singular object?

Reply:
I tested it out on one of our endpoints, and it allowed a singular item and filtered just fine.  Since this is used in an equalTo method, there will only be one value, so I didn't add the parens.